src/components/pages/order/OrderPage.js

```diff
@@ -4,18 +4,35 @@ import styled from "styled-components";
 import { theme } from "../../../theme";
 import logoOrange from "../../../assets/logo-orange.png";
 import Menu from "./main/Menu";
+import { useState } from "react";
+import GlobalContext from "../../../context/GlobalContext";
 
 export default function OrderPage() {
   const { name } = useParams();
+  const [isModeAdmin, setIsModeAdmin] = useState(false);
+  const [panelTabIndex, setPanelTabIndex] = useState(2);
+  const [isPanelReduce, setIsPanelReduce] = useState(false);
+
+  const contextValue = {
+    isModeAdmin,
+    updateIsModeAdmin: setIsModeAdmin,
+    panelTabIndex,
+    setPanelTabIndex,
+    isPanelReduce,
+    setIsPanelReduce,
+  };
 
   return (
-    <OrderPageStyled>
-      <NavBar userName={name} />
-      <main className="main-container">
-        {/* <div className="sideBar-left"></div> */}
-        <Menu />
-      </main>
-    </OrderPageStyled>
+    <GlobalContext.Provider value={contextValue}>
+      <OrderPageStyled>
+        <NavBar userName={name} />
+        <main className="main-container">
+          {/* TODO: basket container */}
```

Comment:
```suggestion
          {/* @TODO: basket container */}
```

C'est la coutume chez les dev qui veut ça.